kernel/init.c

```diff
@@ -259,6 +259,7 @@ static void bg_thread_main(void *unused1, void *unused2, void *unused3)
 void __weak main(void)
 {
 	/* NOP default main() if the application does not provide one. */
+	arch_nop();
```

Comment:
Could we just have `/* Empty by default, will be overwritten by application */` or something similar here?

Reply:
the comment ? or you mean, remove arch_nope() and left this comment ?